### PR TITLE
Improve Readme.md

### PR DESCRIPTION
**Pull request by venkateshwarans**
To make it more clear that this is an npm package and not a starter template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,13 @@ You must make 3 changes to your project to get started:
 
 Check out this [demo project](https://github.com/beyonk-adventures/now-sapper-demo) that uses this builder. It can be used as a template, or a way to verify correct usage of the following instructions.
 
-##### 1. Configure `vercel-sapper` as builder in `vercel.json`
+##### 1. Install `vercel-sapper` as part of `devDependencies`
+
+```bash
+ npm install -D vercel-sapper
```
Comment (antony) [inline README.md]: You don't need to install the builder as a dev dependency. If you do so, Vercel will simply ignore it and install its own copy from npm.

**Review by antony — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (venkateshwarans) [reply README.md]: Thanks. Did not know that. I have updated it.

File: README.md [hunk 2]
```diff
@@ -25,6 +26,7 @@ Check out this [demo project](https://github.com/beyonk-adventures/now-sapper-de
   "builds": [{ "src": "package.json", "use": "vercel-sapper" }]
 }
 ```
+Note: You don't have to install it as any dependency in your project. Vercel build process will install it from npm. More about [builds](https://vercel.com/docs/configuration#project/builds).
```
Comment (antony) [inline README.md:29]:
```suggestion
Note: Don't install `vercel-sapper` it as a dependency in your project. The Vercel build process will automatically install it from npm. More about [builds](https://vercel.com/docs/configuration#project/builds).
```

**Review by antony — COMMENTED** [review]
I'm not sure how in favour I am of a README that tells you **not** to install things. You shouldn't install React either, but we don't really want to write it here.

Seems like we're making the docs longer and providing more for the user to have to read in order to stop them doing something we didn't tell them to do in the first place :)

**Comment by thgh** [comment]
Thanks for the PR, but I don't think it's an improvement.

**Pull request closed without merging**